hawc/apps/common/views.py

```diff
@@ -791,3 +791,20 @@ def wrapper(request, *args, **kwargs):
         return func(request, *args, **kwargs)
 
     return wrapper
+
+
+class FormsetConfiguration:
```

Comment:
add type annotations or at least type comments? can/should this be a [dataclass](https://docs.python.org/3/library/dataclasses.html)?

Reply:
done. dataclass thing is cool, hadn't seen that, switched to it.